### DISCUSSION TITLE
Beta is full

### DISCUSSION THREAD

**Discussion by erlend-sh** [damus-io/damus#290] [General]
Hey 👋 I’m trying to demo the app but TestFlight reports “Beta is full”.

p.s. any interest in setting up a Discord?

**Comment by starbackr-dev** [comment]
Any and all Nostr related comms should happen in github or Nostr client/relays... No more using Twitter, discord, telegram and other competing clients..!

**Reply by erlend-sh** [reply]
I don’t think of discord or telegram as competing clients of nostr. These are group messaging apps specifically made to facilitate coordination.

**Comment by Yzord** [comment]
Although i know it says the beta is full, but is there another way to use this app?

**Comment by jb55** [comment]
you could compile and run it
